### PR TITLE
DOC: Add content to the lesson's overview page

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add content to the lesson's overview page:
- Introduce the lesson and its target audience.
- Add the prerequisites.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@kaitj @josephmje do you know if there is a way to know if the page builds correctly/it renders as expected? Is there a CI that can be set up for the site?

The above PR assumes that a set up and instructor's guide or notes will be added in separate PRs in the near future. Shall we go on and add these? Maybe two initial PRs that can be later improved? Should it be specific to this lesson or shared across existing ones? Pinging @jerdra.

Also, I've observed that the `README` page's title, short explanation and  `About the lesson` paragraph are shared across the neuroimaging lessons. Is this intended?

**Comment by josephmje** [comment]
There is no CI set up for the site but this could be an interesting way of previewing the pages. As of now, there are a couple different ways to build the site:
1.  If you have `ruby` and `bundler` installed on your computer ([instructions here](https://carpentries.github.io/lesson-example/setup.html)), you can run `make lesson-check` in the command line
2. If you don't have those software installed, you can also use a [Docker container](https://github.com/carpentries/lesson-docker) that includes them
3. You can enable GitHub Pages on your own fork so that the site builds at `<gh_username>.github.io/SDC-BIDS-dMRI`

**Comment by jhlegarreta** [comment]
@josephmje thanks for the pointers. I am inclined to try the third option first and see what that gives.

**Comment by jhlegarreta** [comment]
The third option does the job, and the PR contents are rendered correctly :+1:.

Maybe the CI option is something that is worthwhile considering/discussing at some point.

File: index.md [hunk 1]
```diff
@@ -3,15 +3,32 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
+This Data Carpentry lesson aims to introduce learners to the analysis of
+diffusion Magnetic Resonance Imaging (dMRI) data using Python. Its target
```
Comment (kaitj) [inline index.md:7]: This looks good to me - only thing I would change is that the lesson primarily uses Python - there are some exceptions, mainly in the preprocessing (where FSL is used)

File: index.md [hunk 2]
```diff
@@ -3,15 +3,32 @@ layout: lesson
...
+analysis.
 
-<!-- this is an html comment -->
-
-{% comment %} This is a comment in Liquid {% endcomment %}
+> ## Prerequisites for instructors
+>
+> This lesson assumes that the instructors have solid knowledge about diffusion
+> MRI, scientific programming tools used throughout the lesson ([see the
+BIDS-dMRI Setuo page](https://carpentries-incubator.github.io/SDC-BIDS-dMRI/setup.html)),
```
Comment (kaitj) [inline index.md:17]: Looks good - just spelling here: "BIDS-dMRI Setuo page"

**Comment by kaitj** [comment]
> Also, I've observed that the `README` page's title, short explanation and `About the lesson` paragraph are shared across the neuroimaging lessons. Is this intended?

Not sure for the other neuroimaging lesssons. For the dMRI lesson, it had originally started off as a clone of one of the other two lessons, which is why it would share the same paragraphs. Was meaning to update after we had originally "finished" the lessons before we decided to update the lessons using ds000221.

File: index.md (hunk 2 above)
Comment (jhlegarreta) [reply index.md]: Good catch. Fixed in 2f95b90.

File: index.md (hunk 1 above)
Comment (jhlegarreta) [reply index.md]: Specified in 2f95b90. Thanks.

**Comment by jhlegarreta** [comment]
> Not sure for the other neuroimaging lesssons. For the dMRI lesson, it had originally started off as a clone of one of the other two lessons, which is why it would share the same paragraphs. Was meaning to update after we had originally "finished" the lessons before we decided to update the lessons using ds000221.

OK, we'll figure this out once the lessons get to a more complete stage then !

**Comment by josephmje** [comment]
> 
> 
> Overall looks great. Just a couple of minor things with the spelling and that python is primarily used with a few exceptions.
> 
> I guess one thing that I can't recall at the moment, are we making the "SDC-BIDS-IntroMRI" a prerequisite (@josephmje)? I know we have usually taught that first in the past.

Yes, I think we should make the IntroMRI course a prerequisite.

**Comment by jhlegarreta** [comment]
> Yes, I think we should make the IntroMRI course a prerequisite.

Addressed in c134f4a.

**Pull request merged**